### ISSUE TITLE
Cumulative Layout Shift not available?

### ISSUE THREAD

**Issue by cappe** [Carr1005/lighthouse-batch-parallel#15]
Hello,

It seems that there's no Cumulative Layout Shift metric available. The documentation states that customAudits can be used to enable more metrics. However, the link where the documentation points to (https://github.com/GoogleChrome/lighthouse/blob/d8410d5f81db8b3f98304f338afb7309719be0ae/lighthouse-core/config/default-config.js#L365), I cannot find a metric for Cumulative Layout Shift.

Am I missing something or is it just not implemented?

**Comment by Carr1005** [comment]
Hi there!

I found out the Lighthouse version of this module is too old to audit **Cumulative Layout Shift**. Also, the link you mentioned is also outdated, this is the [correct source](https://github.com/GoogleChrome/lighthouse/blob/c2e0e0b37894cb3d1ea520f90581b832fb8d1731/lighthouse-core/config/default-config.js#L428), thanks for pointing it out.

I've tested it, if you are in a rush, clone the project and update the version of the Lighthouse by yarn can directly make your goal. I'll publish the fix asap, really appreciate your feedback. 🙌
